docs/conf.py

```diff
@@ -103,7 +103,7 @@ def resolve_xref(self, env, fromdocname, builder, typ, target, node, contnode):
 html_theme_options = {
     "collapse_navigation": True,
     "display_version": True,
-    "navigation_depth": 5,
+    "navigation_depth": 8,
```

Comment:
Just curious, what does this change?

Reply:
This allows for the navigation to expand more, so the submodules in the apidocs will be more navigable.